brunch-config.js

```diff
@@ -31,8 +31,7 @@ exports.config = {
   plugins: {
     babel: {
       // Do not use ES6 compiler in vendor code
-      ignore: [/^(web\/static\/vendor)/],
-      loose: "all"
```

Comment:
We need to know all versions that loose is meant to support. Also, is it a
binary thing? Or we update Babel or we keep IE8 support? I know we could
list plugins one by one... But that's a no go for me.

On Thursday, November 26, 2015, Eric Meadows-Jönsson <
notifications@github.com> wrote:

> In brunch-config.js
> <https://github.com/phoenixframework/phoenix/pull/1374#discussion_r46008564>
> :
>
> > @@ -31,8 +31,7 @@ exports.config = {
> >    plugins: {
> >      babel: {
> >        // Do not use ES6 compiler in vendor code
> > -      ignore: [/^(web\/static\/vendor)/],
> > -      loose: "all"
>
> I am against removing support for IE8-10 but if you do remove it you
> should also remove the XDomainRequest code.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/phoenixframework/phoenix/pull/1374/files#r46008564>.
>


-- 


*José Valimwww.plataformatec.com.br
<http://www.plataformatec.com.br/>Founder and Director of R&D*